spec/models/availability_spec.rb

```diff
@@ -34,4 +36,31 @@
       expect(availability.partially_available?(event)).to eq(false)
     end
   end
+
+  before(:each) do
+    availability = build(:availability, person: a_person, start_time: start_time, end_time: end_time)
+    availability.save
```

Comment:
I need to persist it to the database since the validations are written that it compares with what is stored. So if I store two availabilities with the same times, only the second throws the validation error.